UICatalog/Scenarios/TopLevelNoWindowBug.cs

```diff
@@ -8,31 +8,31 @@ class TopLevelNoWindowBug : Scenario {
 
 		public override void Run ()
 		{
-			var ntop = new Toplevel (new Rect (0, 0, Application.Driver.Cols, Application.Driver.Rows));
+			Top = new Toplevel (new Rect (0, 0, Application.Driver.Cols, Application.Driver.Rows));
 
 			var menu = new MenuBar (new MenuBarItem [] {
 				new MenuBarItem ("_Файл", new MenuItem [] {
 					new MenuItem ("_Создать", "Creates new file", null),
 					new MenuItem ("_Открыть", "", null),
 					new MenuItem ("Со_хранить", "", null),
-					new MenuItem ("_Выход", "", () => ntop.Running = false )
+					new MenuItem ("_Выход", "", () => Top.Running = false )
```

Comment:
I do not like the use of `Top.Running = false`. When I first wrote this sample, I didn't fully appreciate what it did. In all of our examples we should use `Top.RequestStop` which does the same thing. I would like to remove `Top.Running` as a public API at some point (having public APIs be based on setting a property is just bad design).

Reply:
I agree.